x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/mock/test_providers/test_providers.tsx

```diff
@@ -22,11 +24,24 @@ window.scrollTo = jest.fn();
 /** A utility for wrapping children in the providers required to run tests */
 export const TestProvidersComponent: React.FC<Props> = ({ children }) => {
   const http = httpServiceMock.createSetupContract({ basePath: '/test' });
+  const actionTypeRegistry = actionTypeRegistryMock.create();
+  const mockGetInitialConversations = jest.fn(() => ({}));
+  const mockGetComments = jest.fn(() => []);
+  const mockHttp = httpServiceMock.createStartContract({ basePath: '/test' });
 
   return (
     <I18nProvider>
       <ThemeProvider theme={() => ({ eui: euiDarkVars, darkMode: true })}>
-        <DataQualityProvider httpFetch={http.fetch}>{children}</DataQualityProvider>
+        <AssistantProvider
```

Comment:
What will happen with Assistant functionality if we use data quality without that provider? Do we have a check somewhere?

Reply:
We've got a test for this one: 

https://github.com/elastic/kibana/blob/542f3e5ed5b5df18b4425aa439336e420b4ff542/x-pack/packages/kbn-elastic-assistant/impl/assistant_context/index.test.tsx#L36-L42